src/Containers/packaging/package.csproj

```diff
@@ -28,12 +28,15 @@
                           SetTargetFramework="TargetFramework=$(TargetFramework)"
                           OutputItemType="ContainerLibraryOutput"/>
 
+        <ProjectReference Include="../../Cli/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj"
+                          SetTargetFramework="TargetFramework=$(TargetFramework)"
```

Comment:
@dotnet/sdk-container-builds-maintainers This ends up trying to restore the Utils project for net8.0 but that project currently doesn't include the net8.0 TFM so restore fails. Should this pass in ResolverTargetFramework instead so it builds for net7.0 here or should we add net8.0?  I recall that we build the utility for net7.0 as some of the resolver assemblies load into VSMac which loads those into it's app local net core (which is obviously not net8.0)

CC @dsplaisted 

##[error].dotnet\sdk\8.0.100-preview.4.23260.5\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(266,5): error NETSDK1005: (NETCORE_ENGINEERING_TELEMETRY=Build) Assets file 'D:\a\_work\1\s\artifacts\obj\Tests\Microsoft.DotNet.Cli.Utils\project.assets.json' doesn't have a target for 'net8.0'. Ensure that restore has run and that you have included 'net8.0' in the TargetFrameworks for your project.